rdr_service/api/biobank_specimen_api.py

```diff
@@ -0,0 +1,36 @@
+from flask import request
+
+from rdr_service.api.base_api import UpdatableApi
+from rdr_service.api_util import HEALTHPRO
+from rdr_service.app_util import auth_required
+from rdr_service.dao.biobank_specimen_dao import BiobankSpecimenDao
+from rdr_service.model.utils import from_client_participant_id
+from werkzeug.exceptions import BadRequest
+
+
+class BiobankSpecimenApi(UpdatableApi):
+    def __init__(self):
+        super().__init__(BiobankSpecimenDao(), get_returns_children=True)
+
+    @auth_required(HEALTHPRO)
+    def post(self, p_id):
+        return super().post(participant_id=p_id)
```

Comment:
Do we need POST here? It seems we only support GET and PUT in rdr_service/main.py for BiobankSpecimenApi.

Reply:
You're right, it looks like the design document only calls for PUT as well. I'll remove it.

Reply:
Should I also remove the GET method from this and main.py? The documentation only specifies PUT.